azure-core/azure/core/pipeline_client.py

```diff
@@ -106,14 +118,17 @@ def _request(self, method, url, params, headers, content, form_content, stream_c
 
         return request
 
-    def format_url(self, url, **kwargs):
+    def close(self):
+        self.__exit__()
+
+    def format_url(self, url_template, **kwargs):
         # type: (str, Any) -> str
         """Format request URL with the client base URL, unless the
         supplied URL is already absolute.
 
         :param str url: The request URL to be formatted if necessary.
```

Comment:
I assume the `url` parameter was renamed at some point...

Reply:
Good catch!